### ISSUE TITLE
error: Signing for "RNHTMLtoPDFTests" requires a development team. Select a development team in the project editor. (in target 'RNHTMLtoPDFTests')

### ISSUE THREAD

**Issue by JeremyBradshaw7** [christopherdro/react-native-html-to-pdf#115]
This error occurring on an ios production build. For now have removed all references to RNHTMLtoPDFTests from project.pbxproj, but why is this necessary?

**Comment by dancomanlive** [comment]
Exactly, what the heck is this error ?!

**Comment by dancomanlive** [comment]
Solution:

(XCode) Project > Libraries > RNHTMLtoPDF.xcodeproj > Build Settings > Development Team >
Debug && Release add "Any IOS SDK" on the left side and select your development team on the right side.

**Comment by apamphilon** [comment]
@christopherdro Do you have any plans to add a fix in for this?

**Comment by ryekerjh** [comment]
@dancomanlive When I go into `Libraries >  RNHTMLtoPDF.xcodeproj > Build Settings > ` I cannot find Signing. Is there a way to add it or find it? The options I see are Deployment, Linking, Packaging, Search Paths, and User-Defined. Thanks in advance!

**Comment by kashifaliquazi** [comment]
Does anyone resolved this issues.
We are stuck here and need to make build on urgent basis.
@dancomanlive We also tried added 
(XCode) Project > Libraries > RNHTMLtoPDF.xcodeproj > Build Settings > Signing > Development Team

Still facing the same issue.

Any help would be appreciable.

Thanks

**Comment by ryekerjh** [comment]
@kashifaliquazi I found that I needed to do the above suggestions with Build Settings on your specific package, but also, I needed to switch my build system in xCode to the Legacy Build System (Xcode --> Preferences --> dropdown switch to Legacy Build System)

**Comment by kashifaliquazi** [comment]
Thanks for the quick response @ryekerjh.

But I tried above mentioned steps nothing is helping me I am still getting:

Signing for "RNHTMLtoPDFTests" requires a development team. Select a development team in the project editor.
Code signing is required for product type 'Unit Test Bundle' in SDK 'iOS 12.1'

![Screen Shot 2019-09-20 at 11 32 44 AM](https://user-images.githubusercontent.com/9718787/65303048-7c65be80-db9a-11e9-8dce-a77ea7f86142.png)

We also tried changing build setting to legacy systerm. (but from project settings not from prefrences )
![Screen Shot 2019-09-20 at 11 30 29 AM](https://user-images.githubusercontent.com/9718787/65302944-46c0d580-db9a-11e9-9115-63a2c6eeae51.png)


Still struggling with this issues

Thanks
